pyresample/geometry.py

```diff
@@ -296,11 +296,12 @@ def get_boundary_lonlats(self):
         return (SimpleBoundary(s1_lon.squeeze(), s2_lon.squeeze(), s3_lon.squeeze(), s4_lon.squeeze()),
                 SimpleBoundary(s1_lat.squeeze(), s2_lat.squeeze(), s3_lat.squeeze(), s4_lat.squeeze()))
 
-    def get_bbox_lonlats(self, frequency: Optional[int] = None, force_clockwise: bool = True) -> tuple:
+    def get_bbox_lonlats(self, bbox_shape: Optional[int] = None, force_clockwise: bool = True,
+                         frequency: Optional[int] = None) -> tuple:
         """Return the bounding box lons and lats.
 
         Args:
-            frequency:
+            bbox_shape (formerly frequency):
```

Comment:
`shape`? Also I don't think you can put a parenthetical in the docstring like this as sphinx will treat it like a type declaration. Also shape usually refers to a numpy shape with `(row, column)`. Should this rename maybe reflect that this is a number of points? I got an email that @ghiggi commented the same thing, but I don't see it on here anymore. `num_points`?